media/css/m24/components/navigation-refresh.scss

```diff
@@ -12,525 +12,555 @@
     }
 }
 
-.moz24-navigation-refresh {
+.m24-navigation-refresh {
+    background-color: $m24-color-medium-gray;
     display: flex;
     width: 100%;
     position: sticky;
```

Comment:
**Question/issue:** The existing nav / JS being ported suggests that the nav should only be sticky when `prefers-reduced-motion` is not set, but in the CSS here it's always `position:sticky`? Should the nav just be static at the top of the page if people want to turn animation off?

Happy to punt this to a separate bug / issue as a followup if it helps?